### PR TITLE
Replace iptables with firewall-cmd

### PR DESCRIPTION
**Pull request by dzhengfy**
iptables is a low level command. And firewall-cmd is more convenient for
management which can compatible with more low level commands. So it is
more preferable.

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

File: virttest/utils_firewall_cmd.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+from avocado.utils import process
+from avocado.core import exceptions
+
+from virttest import remote
+from virttest.compat_52lts import decode_to_text
+from virttest.staging import service
+
+
+class Firewall_cmd(object):
```
Comment (balamuruhans) [inline virttest/utils_firewall_cmd.py]: @dzhengfy I have worked on firewall cmd API, please help review https://github.com/avocado-framework/avocado-vt/pull/1785

File: virttest/utils_firewall_cmd.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        :param params: dict with server details
+        :param cleanup: Boolean value, true to cleanup, false to setup
+        """
+        commands = []
+
+        # check the existing ports in remote or local machine
+        check_service = "systemctl status firewalld"
+        start_service = "systemctl start firewalld"
+        check_ports = "firewall-cmd --list-ports"
+        if params:
```
[the remark this answers is not in the document]
Comment (dzhengfy) [reply virttest/utils_firewall_cmd.py]: @sathnaga , thanks for your suggestion. I changed the variable name, and I hope this is helpful for readability.

**Comment by balamuruhans** [comment]
@sathnaga @dzhengfy I had already implemented APIs in generic way in #1785. we can reuse it instead of reimplementing.

**Comment by dzhengfy** [comment]
@balamuruhans ,yes, we have added comments for #1785, please help check. Thanks.

**Comment by dzhengfy** [comment]
Update base on #1785.
This PR depends on #1785

**Pull request merged**